populus/utils/contracts.py

```diff
@@ -140,19 +140,46 @@ def is_contract_name(value):
     SWARM_HASH_SUFFIX
 )
 
+PUSH20_OPCODE = "73"
+ADDRESS_OPCODE = "30"
+EQ_OPCODE = "14"
+EMBEDDED_ADDRESS_REGEX = (
+    '^' +
+    PUSH20_OPCODE +
+    "[0-9a-z]{40}" +
```

Comment:
Likely `a-f`, not `a-z`: it's hexadecimal.

Reply:
Nice catch!